dist/vuelve.esm.js

```diff
@@ -45,10 +46,30 @@ function vuelve(composable, obj) {
       else { variables[key] = ref(cloneDeep(localObj[key])); }
     });
 
-    if (localComposable.mounted) {
-      onMounted(methods[localComposable.mounted.name]);
+    function convertVuelveSyntaxToVue3Syntax(lifecycleHook) {
+      return "on" + lifecycleHook.charAt(0).toUpperCase() + lifecycleHook.slice(1)
     }
 
+    var vuelveLifecycleHookFunctionNames = [
+      "mounted",
+      'beforeUpdate',
+      'updated',
+      'beforeUnmount',
+      'unmounted',
+      'errorCaptured',
+      'renderTracked',
+      'renderTriggered',
+      'activated',
+      'deactivated',
+      'serverPrefetch' ];
+
+    vuelveLifecycleHookFunctionNames.forEach(function (lifecycleHook) {
```

Comment:
it might be a better idea to precompute this only once, when the module is loaded.